ucca/core.py

```diff
@@ -829,6 +909,14 @@ def layers(self):
     def nodes(self):
         return self._nodes.copy()
 
+    @property
+    def categories(self):
+        return self._categories
```

Comment:
I think this and `refined_categories` should copy the attributes (by calling `dict()` and `list()` respectively), like the other getters